besu/src/main/java/org/hyperledger/besu/cli/subcommands/storage/TrieLogSubCommand.java

```diff
@@ -123,6 +131,78 @@ public void run() {
     }
   }
 
+  @Command(
+      name = "export",
+      description =
+          "This command prunes all trie log layers below the retention threshold, including orphaned trie logs.",
+      mixinStandardHelpOptions = true,
+      versionProvider = VersionProvider.class)
+  static class ExportTrieLog implements Runnable {
+
+    @SuppressWarnings("unused")
+    @ParentCommand
+    private TrieLogSubCommand parentCommand;
+
+    @SuppressWarnings("unused")
+    @CommandLine.Spec
+    private CommandLine.Model.CommandSpec spec; // Picocli injects reference to command spec
+
+    @CommandLine.Option(
+        names = "--trie-log-hash",
+        paramLabel = DefaultCommandValues.MANDATORY_LONG_FORMAT_HELP,
+        description = "The hash of the block you want to export the trie log.",
+        arity = "1..1")
+    private String trieLogHash;
+
```

Comment:
Similar to other export commands, it would be good to include the filename as a command line option

Reply:
Maybe consider using a list of blocks? Perhaps next time we need to update more than one 🤷